### PR TITLE
RHDEVDOCS-5545: Pipelines 1.10.5 Release Notes

### PR DESCRIPTION
**Pull request by eromanova97**
• **Aligned team**: Dev Tools
• **OCP version for cherry-picking**: enterprise-4.10 and later
• **JIRA issue**: [RHDEVDOCS-5545](https://issues.redhat.com/browse/RHDEVDOCS-5545)
• **Preview page**: [Release notes for Red Hat OpenShift Pipelines General Availability 1.10.5](https://62724--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#release-notes-1-10-5_op-release-notes)
• **SME Review**: Completed by @concaf
• **QE review**: Completed by @ppitonak 
• **Peer-review**: Completed by @shipsing

### CODE REVIEW

**Comment by Preeticp** [comment]
@ekristova can you please address [this bug](https://spaces.redhat.com/pages/viewpage.action?spaceKey=OSDOCS&title=Getting+started+with+OpenShift+docs) along with this PR please?

File: modules/op-release-notes-1-10.adoc
Comment (concaf) [inline modules/op-release-notes-1-10.adoc:224]:
```suggestion
* Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this issue. Keep in mind that if the pipeline runs are still too huge after compression then the same error will recur.
```

File: modules/op-release-notes-1-10.adoc
Comment (concaf) [inline modules/op-release-notes-1-10.adoc:222]: one more bugfix was merged in upstream pipelines - https://github.com/tektoncd/pipeline/pull/6861

```
Merge podTemplate specified in pipelineRun.spec.taskRunSpecs[].podTemplate along with pipelineRun.spec.podTemplate instead of only considering the one specified at the taskRunSpecs.
```

Comment (concaf) [reply modules/op-release-notes-1-10.adoc:222]: we should add this as well

File: modules/op-release-notes-1-10.adoc
Comment (piyush-garg) [inline modules/op-release-notes-1-10.adoc:226]: this can be removed.

**Review by concaf — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc
Comment (concaf) [inline modules/op-release-notes-1-10.adoc:219]: we need to add that 1.10.5 will be only available in the `pipelines-1.10` channel on OCP 4.10, 4.11, 4.12, 4.13 but will not be available in the `latest` channel for any OCP version including OCP 4.11

**Review by ppitonak — APPROVED** [review]

**Review by concaf — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc
Comment (shipsing) [inline modules/op-release-notes-1-10.adoc:229]:
```suggestion
* Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this problem. Keep in mind that if the pipeline runs are still too huge after compression, then the same error still recurs.
```

File: modules/op-release-notes-1-10.adoc
Comment (shipsing) [inline modules/op-release-notes-1-10.adoc:231]:
```suggestion
* Before this update, only the pod template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object would be considered for a pipeline run. This update fixes the issue. Now, the pod template specified in the `pipelineRun.spec.podTemplate` object is also considered and merged with the template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object.
```

Comment (shipsing) [reply modules/op-release-notes-1-10.adoc:231]: just for better clarity

**Review by shipsing — CHANGES_REQUESTED** [review]

**Review by shipsing — APPROVED** [review]
LGTM

File: modules/op-release-notes-1-10.adoc
Comment (opayne1) [inline modules/op-release-notes-1-10.adoc:229]:
```suggestion
* Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this problem. Remember that if the pipeline runs are still too huge after compression, then the same error still recurs.
```
Vale flags "keep in mind" but I couldn't find the place in the style guide that mentions it. I would recommend using "remember". 

Also, I am curious, is "huge" the correct terminology here? Or could it be large? Only reason is I did a quick search of the docs and noticed huge was only used in reference to page size.

File: modules/op-release-notes-1-10.adoc
Comment (opayne1) [inline modules/op-release-notes-1-10.adoc:231]:
```suggestion
* Before this update, only the pod template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object would be considered for a pipeline run. With this update, the pod template specified in the `pipelineRun.spec.podTemplate` object is also considered and merged with the template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object.
```
I would recommend removing the second sentence just from a minimalism perspective.

**Review by opayne1 — COMMENTED** [review]
Made a few more suggestions for you to consider 😃

File: modules/op-release-notes-1-10.adoc
Comment (ekristova) [reply modules/op-release-notes-1-10.adoc]: Hello @opayne1 thank you for the comments, I tried to replace the word "huge" for RNs before, but the engineer wanted me to use the word "huge" so that is why I used the terminology here.

For now, since the word is already used in our documentation ([here](https://docs.openshift.com/container-platform/4.13/cicd/pipelines/op-release-notes.html#release-notes-1-9-3_op-release-notes)) and we need to release this as soon as possible, I would prefer if we could merge it, if you are okay with it. But next week I will discuss it with the engineers and see if I can replace the word.

I integrated the rest of the changes so I hope it is ready for merge. Thank you!

**Comment by opayne1** [comment]
@ekristova I can merge this if you are ready now

**Comment by ekristova** [comment]
> @ekristova I can merge this if you are ready now

Yes, I was just waiting for the preview to get updated, everything seems good to go now, thank you!

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Comment by opayne1** [comment]
@ekristova you will need to do a manual CP of versions 4.10 and 4.11

**Comment by ekristova** [comment]
For 4.11 the CP is ready, for 4.10 the conflict was expected as we are adding the 1.10 support and therefore the whole RN 1.10 section, so it will take a minute to CP. I will try my best to finish it as soon as possible, I will let you know once that one is done.

**Comment by ekristova** [comment]
@opayne1 the manual CPs are ready and linked here. Thank you!

**Pull request merged**
